js/output/pjs/pjs-resource-cache.js

```diff
@@ -7,14 +7,12 @@ function PJSResourceCache(options) {
     // Insert the images into a hidden div to cause them to load
     // but not be visible to the user
     if (!this.imageHolder) {
-        this.imageHolder = $("<div>")
-            .css({
-                height: 0,
-                width: 0,
-                overflow: "hidden",
-                position: "absolute"
-            })
-            .appendTo("body");
+        this.imageHolder = document.createElement("div");
+        this.imageHolder.style.height = 0;
+        this.imageHolder.style.width = 0;
+        this.imageHolder.style.overflow = "hidden";
+        this.imageHolder.style.position = "absolute";
```

Comment:
I wonder if `Object.assign(this.imageHolder.style, {height: 0, width: 0, overflow: "hidden", position: "absolute"})` would work.  Since this works and is already written let's go with it.

Reply:
That does seem to work, just tried here:
http://jsbin.com/lapisetaki/edit?html,output
Don't know if it'd work in all browsers but interesting to see.
I'll leave it like this, as you say.